pkg/monitoring/rules/alerts/node_alerts.go

```diff
@@ -0,0 +1,56 @@
+package alerts
+
+import (
+	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/ptr"
+)
+
+const (
+	severityAlertLabelKey     = "severity"
+	healthImpactAlertLabelKey = "operator_health_impact"
+)
+
+func nodeAlerts() []promv1.Rule {
+	return []promv1.Rule{
+		{
+			Alert: "NodeHighMemoryUtilization",
+			Annotations: map[string]string{
+				"description": "Memory utilization at {{ $labels.instance }} is very high, exceeding 95% of total memory, including swap. Ensure adequate memory resources are available to prevent performance degradation.",
```

Comment:
I'm not convinced by the wording `exceeding 95% of total memory, including swap.`. It's not clear that memory used + used swap exceeds 95% of the memory.

It may generate confusion that "Memory utilization" "exceeding 95%" of memory + swap which you are calling "total memory"

Reply:
Not sure I understand the issue, but how is 
```
Memory utilization at {{ $labels.instance }} is very high, with used memory and used swap 
exceeding 95% of total memory.
```
If it also doesn't convince you, can you please suggest wording?

Reply:
maybe something like

Memory utilization (RAM + Swap) at {{ $labels.instance }} is very high, 
exceeding 95% of total usable RAM.

(took some working of the values meaning from https://access.redhat.com/solutions/1823)

Reply:
Updated, thanks.